go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/aws/aws-sdk-go v1.40.35 // indirect
 	github.com/bflad/tfproviderlint v0.28.1
 	github.com/client9/misspell v0.3.4
-	github.com/cloudflare/cloudflare-go v0.31.0
```

Comment:
please remove these; we don’t bump version dependencies in the same PR. 

this is also already in master.